server/core/config.cc

```diff
@@ -1700,6 +1711,32 @@ handle_global_item(const char *name, const char *value)
 
             gateway.users_refresh_time = users_refresh_time;
         }
+        else if (strcmp(name, CN_WRITEQ_HIGH_WATER) == 0)
+        {
+            char* endptr;
```

Comment:
You could use here
```
gateway.writeq_high_water = get_suffixed_size(value);
```
which would allow the high and low water marks to be specified as described here: https://github.com/mariadb-corporation/MaxScale/blob/2.2/Documentation/Getting-Started/Configuration-Guide.md#sizes

Should there be a minimum value for the high water mark?